src/routes.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+
+import App from './App';
+import Login from './containers/Login/';
+import SignUp from './containers/SignUp/';
+
+const Routes = (props) =>
+  <Router {...props}>
+    <Switch>
+      <Route path={'/'} exact render={(props) => <App {...props}/>}/>
+      <Route path={'/login'} exact render={(props) => <Login {...props}/>}/>
+      <Route path={'/signup'} exact render={(props) => <SignUp {...props}/>}/>
+    </Switch>
+  </Router>
+
+;
+
+export default Routes;
```

Comment:
Hey @davenjeru, good job on the routes. How did you arrive at the idea of having a file called routes? This could be done in a better way so that we don't have to render two things in the index.js: Normally the app takes all the routes and becomes the base component. Please let's have a discussion on why you thought of that.